tests/queries/0_stateless/02731_system_zookeeper_connection.sql

```diff
@@ -0,0 +1,13 @@
+-- Tags: system_zookeeper_connection table
+
+DROP TABLE IF EXISTS test_zk_connection_table;
+
+CREATE TABLE test_zk_connection_table (
+    key UInt64
+)
+ENGINE ReplicatedMergeTree('/clickhouse/{database}/02731_zk_connection/{shard}', '{replica}')
+ORDER BY tuple();
+
+select count(*)  from system.zookeeper_connection where name='default_zookeeper';
```

Comment:
We can select some columns like host, port, is_expired (it's expected to print something like `localhost, 9181, 0`)

Reply:
correct, can select theses